File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Ask the user for a height between 1-8
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // Print the piramid
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n - 1 - i; j++)
        {
            printf(" ");
        }
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // Change of line
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
# Ask the user for a height between 1-8
while True:
    n = input("Height: ")
    try:
        n = int(n)
        if n >= 1 and n <= 8:
            break
    except:
        continue

# Print the piramid
for i in range(n):

    for j in range(n - 1 - i):
        print(" ", end="")

    for j in range(i + 1):
        print("#", end="")

    # Change of line
    print()
